content/docs/deployment/csminstallationwizard/_index.md

```diff
@@ -69,23 +69,25 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
 
 3. Look over all the fields in the generated `values.yaml` and fill in/adjust any as needed.
 
-4. When the driver is installed using values generated by installation wizard, then the user needs to update the secret for driver by patching the MDM keys, as follows:
+4. When the PowerFlex driver is installed using values generated by installation wizard, then the user needs to update the secret for driver by patching the MDM keys, as follows:
 
 **Steps**
 * `echo -n '<MDM_IPS>' | base64`
 * `kubectl patch secret vxflexos-config -n vxflexos -p "{\"data\": { \"MDM\": \"<GENERATED_BASE64>\"}}"`
 
-5. For the Observability module, please refer [Observability](../../observability/deployment/#post-installation-dependencies) to install the post installation dependencies.
+5. If Observability is enabled, please refer to [Observability](../../observability/deployment/#post-installation-dependencies) to install the post installation dependencies.
 
-6. If Authorization is enabled , please refer to [Authorization](../../authorization/deployment/helm/) for the installation and configuration of the Proxy Server.
+6. If Authorization is enabled, please refer to [Authorization](../../authorization/deployment/helm/) for the installation and configuration of the Proxy Server.
 
 >NOTE: Only the Authorization sidecar is enabled by the CSM Installation Wizard. The Proxy Server has to be installed and configured separately.
 
-7. If the Volume Snapshot feature is enabled, please refer to [Volume Snapshot for PowerStore](../../csidriver/installation/helm/powerstore/#optional-volume-snapshot-requirements) and [Volume Snapshot for PowerMax](../../csidriver/installation/helm/powermax/#optional-volume-snapshot-requirements) to install the Volume Snapshot CRDs and the default snapshot controller.
+7. If Replication is enabled, please refer to [Replication](../../replication/deployment/) to configure communication between Kubernetes clusters.
```

Comment:
change to "If Replication is enabled, please refer to [Replication](../../replication/deployment/) on how to configure communication between Kubernetes clusters"